src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -18,6 +18,8 @@
 
 package me.ryanhamshire.GriefPrevention;
 
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
```

Comment:
imma assume this is cuz spigot/cb uses these, that we don't have to do anything to our pom?

Reply:
https://github.com/GriefPrevention/GriefPrevention/issues/2212#issuecomment-1942239304

spigot-api uses it. mk.

Reply:
Yeah, Spigot provides Guava, Lang3, and a couple other libraries.